go/test/endtoend/filelock/filelock_unix.go

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build darwin || dragonfly || freebsd || illumos || linux || netbsd || openbsd
```

Comment:
Why not use `!windows` here too?

Reply:
@mattlord These are straight copied files from a Go internal package, so I've kept them as is.